vyxal/elements.py

```diff
@@ -110,15 +110,15 @@ def process_element(
     "Q": process_element("exit()", 0),
     "R": (
         """
-ts = (vy_type(stack[-1]),) if len(stack) < 2 else (vy_type(stack[-2]), vy_type(stack[-1]))
+rhs, lhs = pop(stack, 2, ctx)
```

Comment:
I'm not particularly keen on doing it this way because taking two inputs moves the input cycle by an extra input if only one value is needed.

Reply:
`ctx.inputs[-1][1] -= 1`. But that's only if input was needed. Obviously don't move it back if no input was taken.

Note that the indexing might be wrong. Check the specs in context.py (I'll make sure to do so myself in my next review)

Reply:
But it pushes that input to the stack anyway so it won't matter right?

Reply:
It pushes it to the stack, but the counter that keeps track of which implicit input is next is also incremented.